rails/config/initializers/delayed_web.rb

```diff
@@ -0,0 +1,4 @@
+# Tell Delayed::Web that we're using ActiveRecord as our backend.
+Rails.application.config.to_prepare do
+  Delayed::Web::Job.backend = 'active_record'
+end
```

Comment:
I thought this was removed?

Reply:
This is for the replacement gem.  We replaced `delayed_job_web` with `delayed-web` and it requires this new configuration fie.